catalyst/utils/typing.py

```diff
@@ -1,12 +1,23 @@
-from typing import Union  # isort:skip
+from typing import Union, Dict  # isort:skip
 
 import torch
 from torch import nn, optim
+from torch.utils import data
 
-Model = nn.Module
+Model = Union[nn.Module, Dict[str, nn.Module]]
 Criterion = nn.Module
-Optimizer = optim.Optimizer
+Optimizer = Union[optim.Adam, optim.SGD]
```

Comment:
but, why?

Reply:
just use `Optimizer = optim.Optimizer`

Reply:
PyCharm showed to me that there is no such reference in torch.optim. Ill investigate this more.

Reply:
Probably i've been confused by it, so i'll return it back as it was.